### DISCUSSION TITLE
Cannot add attachment

### DISCUSSION THREAD

**Discussion by r00tr4v3n** [dani-garcia/vaultwarden#4326] [Q&A]
Hello,
I don't see any attachment buttons in secure notes. (v1.30.3 )
Domain configuration matches. Any configuration I'm missing?

**Comment by BlackDex** [comment]
See: https://bitwarden.com/help/attachments/

**Reply by r00tr4v3n** [reply]
I did. I don't see the dotted options?

**Reply by stefan0xC** [reply]
![Screenshot 2024-02-08 at 20-19-04 Vaults Vaultwarden Web](https://github.com/dani-garcia/vaultwarden/assets/509385/6d5faca1-7deb-4f87-91c5-39eba502a765)

**Reply by r00tr4v3n** [reply]
Thank you! I did not get one has to create an (empty) note first to attach something... That's not very convenient.

**Reply by kpfleming** [reply]
Bitwarden doesn't support 'files' as a top-level object which can be stored in the vault, but does allow them to be attached to other objects (notes, logins, identities, etc.). Thus, since they are attachments, they have to be attached to something.

**Reply by r00tr4v3n** [reply]
Yes, but attaching them at creation time would be nice instead of having to save an empty note.

**Reply by BlackDex** [reply]
Would be nice indeed. But not something this project can fix.
